ts/sagas/backendInfo.ts

```diff
@@ -37,6 +37,7 @@ function* backendInfoWatcher(): IterableIterator<Effect> {
       typeof getServerInfo
     > = yield call(getServerInfo, {});
     if (
+      backendInfoResponse &&
```

Comment:
is this necessary? `getServerInfo` always returns a `Promise`...

Reply:
ah I see, the catch block resolves to `undefined`, you should also update the return type of `getServerInfo` then

Reply:
actually, the correct solution would be to resolve the promise to a `Left` at row 31

Reply:
Fix in a794d68